app/controllers/katello/api/v2/host_packages_controller.rb

```diff
@@ -87,14 +88,17 @@ def resource_class
 
     private
 
-    def find_host
-      @host = resource_finder(::Host::Managed.authorized("edit_hosts"), params[:host_id])
+    def find_host_with_facet
       fail HttpErrors::NotFound, _("Couldn't find host with host id '%s'") % params[:host_id] if @host.nil?
       fail HttpErrors::NotFound, _("Host id '%s' does not have a content facet") % params[:host_id] if @host.content_facet.nil?
       fail HttpErrors::NotFound, _("Host id '%s' content facet is missing a uuid") % params[:host_id] if @host.content_facet.uuid.nil?
       @host
     end
 
+    def find_host
+      @host = resource_finder(::Host::Managed.authorized("edit_hosts"), params[:host_id])
```

Comment:
I know this was just moved as part of this PR but should it be "read_hosts" instead of "edit_hosts"?